Sources/NIO/ByteBuffer-core.swift

```diff
@@ -708,6 +708,7 @@ public struct ByteBuffer {
     ///
     /// - parameters:
     ///     - minimumCapacity: The minimum capacity that will be (re)allocated for this buffer
+    @available(*, deprecated, message: "Use an `Int` as the argument")
     public mutating func clear(minimumCapacity: _Capacity) {
         if !isKnownUniquelyReferenced(&self._storage) {
```

Comment:
Can we just forward this implementation to the new one?

Reply:
I wasn't sure how - in this case `renamed` doesn't work as the functions have the same name. Any suggestions?

Reply:
Oh I meant just call `self.clear(minimumCapacity: Int(minimumCapacity))`

Reply:
Ah! Sure